src/tests/ftest/util/mpio_utils.py

```diff
@@ -27,6 +27,9 @@
 import subprocess
 import paramiko
 import socket
+from general_utils import exports_cmd
```

Comment:
(pylint-no-name-in-module) No name 'exports_cmd' in module 'general_utils'
(pylint-unused-import) Unused exports_cmd imported from general_utils